home_robot_msgs/ObjectLocation object_location
---
home_robot_msgs/NavTuple placements